pkg/operator/staticresourcecontroller/static_resource_controller.go

```diff
@@ -82,6 +86,16 @@ func NewStaticResourceController(
 	return c
 }
 
+// IgnoreNotFoundOnCreate makes the controller to ignore NotFound errors when applying a manifest.
+// Such error is returned by the API server when the controller tries to apply a CR for CRD
+// that has not yet been created.
+// This is useful when creating CRs for other operators that were not started yet (such as ServiceMonitors).
+// NotFound errors are reported in <name>Degraded condition, but with Degraded=false.
+func (c *StaticResourceController) IgnoreNotFoundOnCreate() *StaticResourceController {
+	c.ignoreNotFoundOnCreate = true
+	return c
+}
```

Comment:
nit: since it's a constructor, what about prefixing it with `With`?

Reply:
Renamed to `WithIgnoreNotFoundOnCreate`.